Makefile

```diff
@@ -23,6 +23,9 @@ build-functests: bazel-build-functests
 bazel-build-image-bundle:
 	hack/dockerized "export BUILD_ARCH=${BUILD_ARCH} && hack/bazel-fmt.sh && DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} IMAGE_PREFIX=${IMAGE_PREFIX} hack/bazel-build-image-bundle.sh"
 
+multi-arch-build:
```

Comment:
here an alternative possibility to this approach:

You could modify `hack/bazel-push-images.sh` a little bit to keep the digests of each architecture.

E.g. changing

```bash
rm -rf ${DIGESTS_DIR}
mkdir -p ${DIGESTS_DIR}

for f in $(find bazel-bin/ -name '*.digest'); do
    dir=${DIGESTS_DIR}/$(dirname $f)
    mkdir -p ${dir}
    cp -f ${f} ${dir}/$(basename ${f})
done
```

to

```
DIGEST_DIR=${DIGESTS_DIR}/${ARCHITECTURE}

rm -rf ${DIGESTS_DIR}
mkdir -p ${DIGESTS_DIR}

for f in $(find bazel-bin/ -name '*.digest'); do
    dir=${DIGESTS_DIR}/$(dirname $f)
    mkdir -p ${dir}
    cp -f ${f} ${dir}/$(basename ${f})
done
```

or something along that line.


We could have the following structure in `_out/digest` (the current DIGEST_DIR location):

 * _out/digests/entrypoints
 * _out/digests/arm64
 * _out/digests/amd64
 
 If `hack/bazel-push-images.sh` would be called WITH a list of architectures to build, it would do the following:
  * build arm64 and safe digests in _out/digests/arm64
  * build amd64 and safe digests in _out?digests/amd64
  * run docker manifest, and save the resulting manifest digests in _out/digests/entrypoint
  * The current manifest render command would then pick up the digests from _out/digests/entrypoint
 
 If `hack/bazel-push-images.sh` would be called WITHOUT an architecture list, it would do the following:
  * build the current architecture, like right now and safe it in _out/digests/{arch}
  * then simply copy the content from _out/digests/{arch} to _out/digests/entrypoint
  * The current manifest render command would then pick up the digests from _out/digests/entrypoint

We would basically keep all the push logic on one place and the follow-up scripts which rely on the digest folder would just pick up digests as before and would not see any difference.

Reply:
Make sense.  I incorporated a bunch of this.